elixir/bank-account/lib/account.exs

```diff
@@ -0,0 +1,50 @@
+defmodule Account do
+  use GenServer
+
+  #Client
+  def open_bank do
+    with {:ok, pid} <- GenServer.start_link(__MODULE__, %{balance: 0, status: :open}) do
+      pid
+    else
+      error -> raise "Account was not opened. #{inspect(error)}"
+    end
```

Comment:
I know I lead you astray here and it is totally my fault.. but..

When you have a `with` statement with a single clause, and only one `else` case... use a case statement instead!


```suggestion
    case GenServer.start_link(__MODULE__, %{balance: 0, status: :open}) do
      {:ok, pid} -> pid
      error -> raise "Account was not opened. #{inspect(error)}"
    end
```

Reply:
Thank you! I changed this one